lib/screenplay_web/router.ex

```diff
@@ -39,6 +39,10 @@ defmodule ScreenplayWeb.Router do
     plug(ScreenplayWeb.EnsurePaMessageAdmin)
   end
 
+  pipeline :enforce_api_key do
+    plug(ScreenplayWeb.Plugs.EnforceApiKey)
```

Comment:
Minor naming thing: The other pipelines and plugs involving auth(entication|orization) all start with `Ensure`. If this is the same kind of thing as those, maybe it could be named `EnsureApiKey`? Or `EnsureApiAuth`, as we are not just ensuring there _is_ an API key in the request, but that it's valid.